panda/src/navigation/navMesh.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NAVMESH_H
+#define NAVMESH_H
+
+#include "Recast.h"
+#include "DetourNavMesh.h"
+#include "pandaFramework.h"
```

Comment:
pandaFramework.h should not be included here

Reply:
Tried removing it. Gave errors.
Probably because "TypedWritableReferenceCount" requires that header.

Reply:
Well, maybe you should replace it with an include to `typedWritableReferenceCount.h` :-)

It may be that pandaFramework.h is indirectly including the header that you actually need.

Reply:
Looks like you added the typedWirtableReferenceCount.h include, so now it should be OK to remove pandaFramework.h.

Reply:
It still gives error